baggage/baggage.go

```diff
@@ -397,6 +405,9 @@ func (m Member) Properties() []Property { return m.properties.Copy() }
 
 // String encodes Member into a header string compliant with the W3C Baggage
 // specification.
+// It would return empty string if the key is invalid with the W3C Baggage
+// specification. This could happen for a UTF-8 key, as it may contain
+// invalid characters.
```

Comment:
Is it not repetitive?

```suggestion
// specification.
```

Reply:
A little bit as I want to remind the users that there is a counterintuitive issue where a UTF-8 key, which is accepted by the `*New` API, may be dropped by the `String` method.